job-tutorial/Dockerfiles/cuda8.0-cudnn6/Dockerfile.build.base

```diff
@@ -76,6 +76,6 @@ ENV HADOOP_PREFIX=${HADOOP_INSTALL} \
     HADOOP_OPTS="-Djava.library.path=${HADOOP_INSTALL}/lib/native"
 
 ENV PATH=/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:${HADOOP_BIN_DIR}:${HADOOP_SBIN_DIR} \
-    LD_LIBRARY_PATH=/usr/local/nvidia/lib:/usr/local/nvidia/lib64:/usr/local/cuda/lib64:${JAVA_HOME}/jre/lib/amd64/server
+    LD_LIBRARY_PATH=/usr/local/nvidia/lib:/usr/local/nvidia/lib64:/usr/local/cuda/lib64:/usr/local/cuda/targets/x86_64-linux/lib/stubs:${JAVA_HOME}/jre/lib/amd64/server
```

Comment:
>/usr/local/cuda/targets/x86_64-linux/lib/stubs [](start = 88, length = 46)

What will happend if without this one in PATH?

Reply:
can not find lib in this path